pkg/virt-handler/vm_test.go

```diff
@@ -1656,7 +1657,7 @@ var _ = Describe("VirtualMachineInstance", func() {
 				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].Name).To(Equal(podCacheInterface.Iface.Name))
 				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].IP).To(Equal(podCacheInterface.PodIP))
 				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].IPs[0]).To(Equal(podCacheInterface.PodIP))
-
+				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].IPs[1]).To(Equal(podCacheInterface.PodIPs[1]))
```

Comment:
I would also a verification the length of `arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].IPs` abd `podCacheInterface.PodIPs` is the same.

Reply:
done